packages/overlays/package.json

```diff
@@ -102,7 +101,7 @@
     "root": true
   },
   "dependencies": {
-    "@sanity/visual-editing": "2.1.8"
+    "@sanity/visual-editing": "workspace:^"
```

Comment:
Using `workspace:^` here is intentional, we want the published version to be `^2.1.8` instead of an exact version, so we can stop publishing new versions of `@sanity/overlays` and remove it from the monorepo.